app/src/main/res/values/dimens.xml

```diff
@@ -541,7 +540,10 @@
 
   <dimen name="promoted_story_card_layout_margin_end">8dp</dimen>
   <dimen name="promoted_story_card_layout_margin_start">8dp</dimen>
-  <dimen name="promoted_story_card_layout_width">280dp</dimen>
+  <!-- The width is set to sp so that it will scale according to the device screen -->
```

Comment:
<!-- The width is set to sp so that it will scale according to the device screen and won't fail for Accessibility Scanner on latest android versions -->

Or something similar to add more context to it.

Reply:
Even if we use sp, accessibility scanner still shows suggestion to improve text scaling. If we set width to wrap_content then only accessiblity scanner doesn't show any improvement but here it's not possible. 

![image](https://user-images.githubusercontent.com/43074241/200378527-a4cc0488-935d-4e17-8749-f2349d502d34.png)

Reply:
Ack.